src/pyshark/capture/pipe_capture.py

```diff
@@ -39,16 +36,28 @@ def get_parameters(self, packet_count=None):
         Returns the special tshark parameters to be used according to the configuration of this class.
         """
         params = super(PipeCapture, self).get_parameters(packet_count=packet_count)
-        params += ['-r', '-']
-        return params
-
-    @asyncio.coroutine
-    def _get_tshark_process(self, packet_count=None):
-        proc = yield From(super(PipeCapture, self)._get_tshark_process(packet_count=packet_count,
-                                                                       stdin=self._pipe))
-        raise Return(proc)
+        params.extend(['-i{}'.format(self._pipe), '-'])
+        return params[:-1]
 
     def close(self):
         # Close pipe
-        self._pipe.close()
+        # self._pipe.close()  # Don't close the pipe. This should be the job of whatever is piping into it.
```

Comment:
That's fine, but remove this line entirely and add this to the class doc in CAPS or otherwise in an obvious manner